src/main/kotlin/fsrss/FeedController.kt

```diff
@@ -1,17 +1,63 @@
 package fsrss
 
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import org.springframework.http.HttpStatus
 import org.springframework.stereotype.Controller
-import org.springframework.web.bind.annotation.GetMapping
-import org.springframework.web.bind.annotation.PathVariable
-import org.springframework.web.bind.annotation.ResponseBody
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.client.HttpClientErrorException
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Path
 
 
 @Controller
 class FeedController {
+    val path: String = System.getenv("RSS_FEEDS_PATH") ?: "/"
```

Comment:
```suggestion
    val path: String = System.getenv("RSS_FEEDS_PATH") ?: "/data"
```
To match my just written readme. Also easier for much needed security at some point I guess.